index.js

```diff
@@ -1,129 +1,614 @@
-module.exports = babel => {
-  const t = babel.types
+'use strict'
 
-  return {
-    inherits: require('babel-plugin-syntax-jsx'),
-    visitor: {
-      Program(path) {
-        path.traverse({
-          JSXOpeningElement(path) {
-            const vModelAttribute = path.node.attributes.filter(node => node.name.name === 'v-model')[0]
+var allowedModifiers = ['trim', 'number', 'lazy']
+var RANGE_TOKEN = '__r'
+var CHECKBOX_RADIO_TOKEN = '__c'
 
-            if (!vModelAttribute) {
-              return
-            }
+var htmlTags = [
```

Comment:
is there a way that you can extract all of this elements into separate json file like:
https://github.com/vuejs/eslint-plugin-vue/blob/master/lib/utils/html-elements.json